src/editor/sprite_editor/sprite_editor_frames.cpp

```
@@ -0,0 +1,270 @@
+                        if (step.frameIndex >= frameToDelete && step.frameIndex > 0) {
+                            --step.frameIndex;
+                        }
+                    }
+                }
+                PushFrameClipAction(std::move(beforeFrames), std::move(beforeClips),
+                                    beforeSel, m_selectedFrame);
+            }
+
+            // ---- Selected frame mini-preview (with pivot drag) ----
+            ImGui::TableSetColumnIndex(1);
+            auto const* image = m_spriteSheet->GetImage().get();
```

Clang (14.0.6)

[error] variable 'image' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)